### DISCUSSION TITLE
提供网络数据解析连接ID的接口

### DISCUSSION THREAD

**Discussion by jiangema** [Tencent/tquic#429] [General]
由于单线程处理能力限制。需要开启多线程处理。
现在得做法是
1、开启一个线程专门读取udp网络数据
2、开启多个线程，每个线程拥有一个quic Endpoint。
3、网络数据读取线程将网络数据按照来源ip:prot映射到对应的quic数据处理线程。客户端连接迁移后，需要按连接id将
网络数据映射到正确的quic数据处理线程。
4、目前获取网络数据的连接ID函数，使用的是packet::PacketHeader::from_bytes。 packet是私有的，现在临时的解决方法是将packet改为公有的。
希望能提供一个api 获取网络数据对应的连接ID的接口，或者是将packet mod导出为公有的。

**Comment by iyangsj** [comment]
`PacketHeader`类型在`lib.rs`已导出为公有
https://github.com/Tencent/tquic/blob/develop/src/lib.rs#L1224

可以直接使用 `tquic::PacketHeader::from_bytes()` (而不是`tquic::packet::PacketHeader::from_bytes()`)
https://docs.rs/tquic/latest/tquic/struct.PacketHeader.html#method.from_bytes
